### ISSUE TITLE
In Python3 here should be converted into number by tensor.item()

### ISSUE THREAD

**Issue by ShinChoon** [buaabai/Ternary-Weights-Network#6]
Just a note for others
https://github.com/buaabai/Ternary-Weights-Network/blob/ccc7f94eb325a624fae58ebed38cdd6106612251/main.py#L107

https://github.com/buaabai/Ternary-Weights-Network/blob/ccc7f94eb325a624fae58ebed38cdd6106612251/main.py#L120

`loss.data[0]` -> `loss.item()`
`test_loss += criterion(output, target).data[0]` -> `test_loss += criterion(output, target).item()`
